src/main/java/DAO/DBConnection.java

```diff
@@ -0,0 +1,42 @@
+package dao;
+
+import dao.exceptions.DAOException;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DbConnection {
+    private static DbConnection instance;
+    private static DbAccess access;
+
+    public DbConnection() {
+        getInstance();
+    }
+
+    public DbConnection(String properties) {
+        getInstance(properties);
+    }
```

Comment:
FYI iconical singleton pattern uses private constructor.
But for this project it's okay